packages/cms-cli/commands/hubdb.js

```diff
@@ -105,8 +109,79 @@ function configureHubDbFetchCommand(program) {
   addConfigOptions(program);
 }
 
+function configureHubDbClearCommand(program) {
+  program
+    .version(version)
+    .description('Clear all rows in a HubDB table')
+    .arguments('<tableId>')
+    .action(async (tableId, command = {}) => {
+      setLogLevel(command);
+      logDebugInfo(command);
+      const { config: configPath } = command;
+      loadConfig(configPath);
+      checkAndWarnGitInclusion();
+
+      if (!(validateConfig() && (await validatePortal(command)))) {
+        process.exit(1);
+      }
+      const portalId = getPortalId(command);
+      try {
+        const draftTable = await clearHubDbTableRows(portalId, tableId);
+        const deletedRowCount = draftTable[0].rowIds.length;
+        if (deletedRowCount > 0) {
+          logger.log(
+            `Removed ${deletedRowCount} rows from HubDB table ${tableId}`
+          );
+          const { rowCount } = await publishTable(portalId, tableId);
+          logger.log(`HubDB table ${tableId} now contains ${rowCount} rows`);
+        } else {
+          logger.log(`HubDB table ${tableId} is already empty`);
+        }
+      } catch (e) {
+        logger.error(e);
```

Comment:
I suggest using `logErrorInstance` for this because it has some extra stuff that makes the output more friendly. We need to do the same for the other hubdb commands. See https://github.com/HubSpot/hubspot-cms-tools/blob/master/packages/cms-lib/errorHandlers.js#L241

Reply:
Ah! Gotcha, I swapped those out. I there are a couple of `logger.log()`s that I kept since they provide more context. Is that ok?